app/knowledge_panels.py

```diff
@@ -210,24 +213,28 @@ async def wikidata_kp(self):
         for id, val in enumerate(entities):
             html.append(
                 {
-                    "id": int(id),
-                    "subtitle": val.description_tag,
-                    "source_url": f"https://www.wikidata.org/wiki/{val.entity_id}",
-                    "elements": [
-                        {
-                            "element_type": "text",
-                            "text_element": val.label_tag,
-                        },
-                        {
-                            "element_type": "links",
-                            "wikipedia": val.wikipedia_relation,
-                            "image_url": val.image_url,
-                            "open_street_map": val.OSM_relation,
-                            "INAO": val.INAO_relation,
-                        },
-                    ],
+                    "element_type": "text",
+                    "text_element": {
+                        "source_label": val.label_tag,
+                        "source_description": val.description_tag,
```

Comment:
source_label and source_description do not seem to exist

Also you must have a "html" in your text_element !

So:
```suggestion
                        "html": f"<p><em>{val.label_tag}</em></p><p><small>{val.description_tag}</small></p>",
```